doc/src/modules/utilities/iterables.rst

```diff
@@ -29,7 +29,13 @@ functions to generate partitions that can be used as low-level tools for
 routines:  ``partitions`` and ``multiset_partitions``. The former gives
 integer partitions, and the latter gives enumerated partitions of elements.
 There is also a routine ``kbins`` that will give a variety of permutations
-of partions.
+of partions. The ``nT`` function from ``sympy.functions.combanitorial.numbers``
+can be used to simply count the number of partions.
+
+See Also
+========
+
+sympy.utilities.iterables.ordered_partitions
```

Comment:
of partitions. And to obtain partitions as a list instead of a dictionary, there
is the ``ordered_partition`` function which is quite fast. Finally, to simply
obtain a count of the number of partitions without enumerating them, there
is the ``nT`` function.

See Also
========
...list all of those mentioned here

Reply:
Thanks for reviewing!
Will make the corrections...